### PR TITLE
save local/from_schemasheets yaml and retire makefile target that builds it

### PR DESCRIPTION
**Pull request by turbomam**

### CODE REVIEW

File: src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml [hunk 1]
```diff
@@ -0,0 +1,1110 @@
...
+    base: float
+    uri: xsd:decimal
+  language code:
+    name: language code
+    description: a string representation of a language
+    base: str
+    uri: xsd:language
+enums:
+  BioticRelationshipEnum:
+    description: placeholder enum descr
```
Comment (pkalita-lbl) [inline src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml:40]: Would it make sense to remove all these placeholder descriptions with this PR?

**Review by pkalita-lbl — APPROVED** [review]
One minor comment. If you want to track that as a separate issue that's fine with me. Either way, feel free to merge when you're ready.

File: src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml (hunk 1 above)
Comment (turbomam) [reply src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml:40]: yes, I thought I had. Maybe that was in some different project.

Comment (turbomam) [reply src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml:40]: - https://github.com/microbiomedata/submission-schema/issues/239

**Pull request merged**